mods/measure/init.lua

```diff
@@ -0,0 +1,50 @@
+local function deltas_as_string(pos1, pos2)
+	return("Deltas: X: "..math.abs(pos1.x-pos2.x).." | Y: "..math.abs(pos1.y-pos2.y).." | Z: "..math.abs(pos1.z-pos2.z))
+	end
+
+minetest.register_craftitem("measure:stick", {
+
+	description = "Measuring Stick",
+	inventory_image = "measure_stick.png",
+	stack_max = 1,
+
+	on_use = function(itemstack, user, pointed_thing)
+
```

Comment:
Since `user:get_player_name()` is used multiple times in this function, you can add the following:
```lua
local name = user:get_player_name()
```

...and replace all other occurrence of `user:get_player_name()` with `name`. Results in lesser API calls and cleaner-looking code.